### PR TITLE
Spelling take two

### PR DESCRIPTION
**Pull request by plaindocs**
A look at spelling using `vale` instead of `sphinxcontrib-spelling`.

Remaining words to fix or  add:

```
    1 Afterwork
      2 alloted
      1 Amaechi
      1 Animatron
      1 annonimised
      1 annouced
      2 Annoyment
      1 anonymised
      1 anually
      1 Appcues
      1 Arvada
      1 Atul
      1 bocoup
      1 Buisness
      1 caniculares
      1 capella
      1 cetera
      1 chidren
      1 Chronologue
      1 Circonus
      1 Clackamas
      2 cofounders
      1 Colab
      1 colorism
      1 comms
      1 communiy
      1 contenet
      1 contributer
      1 coorindator
      5 correspsond
      1 coversation
      3 coworking
      1 Creds
      1 crunchies
      1 Cyrille
      1 Datacoral
      1 debatably
      1 decolonial
      1 deployers
      1 dififcult
      1 discusssion
      1 disdash
      1 disgreement
      1 doclet
      1 docsets
      2 docsteam
      1 doxing
      1 Dururing
      1 ee
      1 embarassment
      1 Envestnet
      1 Esc
      4 espcially
      1 esque
      1 etcetera
      1 everthing
      1 everyones
      2 familar
      1 feckups
      1 findability
      1 finn
      1 flipbooks
      1 flipcharts
      1 foody
      1 freetime
      1 fulltime
      1 gists
      1 gnelson
      1 Greenshot
      1 gullwings
      1 Gymglish
      1 Hackos
      1 Hackup
      2 har
      2 Harbrace
      1 hercule
      1 Herndon
      1 ie
      1 incentivizes
      1 Indispensible
      1 infoarchitecture
      1 informtation
      1 inhouse
      1 Institue
      1 intial
      1 Json
      1 Juneteeth
      1 Karlruhe
      1 Karp
      1 Kata
      1 Klatches
      2 lagering
      1 Lexique
      1 lgbtq
      1 Lidwell
      1 Lightspeed
      1 Linguee
      1 Linkurious
      1 Longo
      1 longwindedness
      1 Lullabot
      1 Lyn
      1 mabl
      1 Macrometa
      1 md
      1 Medio
      1 Mehrotra
      1 Meng
      1 microbuses
      1 mindshare
      1 miniconf
      1 minorly
      1 misalignments
      1 misgendering
      1 mit
      2 mortems
      1 multigendered
      1 mundance
      1 Musem
      1 Ngram
      1 Octover
      1 offtopic
      1 orgs
      1 otherside
      1 Pac
      1 Paik
      1 Panotpo
      1 parallelization
      1 penty
      1 Philosophie
      1 pictoral
      1 plaintext
      1 plusses
      1 Pomerium
      1 practioner
      1 pratice
      1 preconferences
      4 prework
      1 pronous
      2 prospecti
      1 Protip
      1 pushback
      1 Rands
      1 realtime
      2 recored
      1 recruitement
      1 rewatch
      1 roff
      1 rulesets
      1 sanfrancisco
      1 Satismeter
      1 Sawppy
      1 Screenpresso
      1 searchability
      1 Selber
      1 semp
      1 Sketchnoting
      1 Slite
      3 Smale
      1 souvenier
      1 spectrums
      1 standups
      1 Stockhom
      1 Streat
      1 Swarts
      1 Tastypie
      1 technial
      1 telcos
      1 teleporting
      1 Testlodge
      1 testthedocs
      1 thez
      1 thoghts
      1 thon
      1 Thon
      1 thumbsdown
      2 ticketholders
      4 titocode
      1 tl
      1 tldr
      1 Toks
      1 triaging
      1 tv
      1 uals
      1 unicode
      1 unselectable
      1 unzufrieden
      3 Upshur
      1 Upvest
      1 url
      1 Usabilla
      1 Vistar
      1 Vitaly
      1 welcomewagon
      1 Wendt
      1 Wheelly
      1 whitepapers
      1 Whitton
      1 Wilska
      1 wondeful
      1 Writin
      5 writingday
      1 wtdbadge
      2 wurly

```

### CODE REVIEW

**Comment by bethaitman** [comment]
Things I think are not typos:

- capella (a capella)
- caniculares (because I put that in a newsletter, sorry)
- cetera (et cetera)
- cofounders
- comms (tech comms)
- coworking
- debatably
- docsets
- etcetera
- findability
- flipbooks
- flipcharts
- fulltime
- Indispensible
- longwindedness
- md
- mindshare
- miniconf
- minorly
- misalignments
- misgendering
- multigendered
- offtopic
- parallelization
- plaintext
- plusses
- preconferences
- prework
- pushback
- realtime
- searchability
- Sketchnoting
- standups
- ticketholders
- tldr
- triaging
- unicode
- unselectable
- whitepapers

Things I think might not be typos, but depends on context (I got bored adding context sorry):

- Appcues (company name)
- Arvada (place in Colorado)
- Atul (company name)
- Circonus (company name)
- Clackamas (place in Oregon)
- Cyrille (name)
- Datacoral (company name)
- deployers (I think it's a word?)
- decolonial (similar?)
- Envestnet (company name)
- gists
- foody
- freetime
- Greenshot
- Gymglish
- Hackos
- Harbrace
- incentivizes
- infoarchitecture
- inhouse
- Lidwell
-  Lightspeed
- Linguee
- Linkurious
- Longo
- Lullabot
- Lyn
- Macrometa
- Medio
- Mehrotra
- Meng
- mortems
- Paik
- Panotpo
- pictoral
- Pomerium
- rewatch
- rulesets
- Satismeter
- Screenpresso
- telcos
- teleporting
- Testlodge
- testthedocs
- Upvest
- Usabilla
- Vistar
- Vitaly
- welcomewagon

File: vale/Vocab/WTD/accept.txt [hunk 1]
```diff
@@ -0,0 +1,483 @@
...
+webpage
+themself
+walkthrough
+videographer
+workshopping
+Upstreams
+waitlist
+docstrings
+Documententarian
+documnentarians
```
Comment (bethaitman) [inline vale/Vocab/WTD/accept.txt:483]: typo

File: vale/Vocab/WTD/accept.txt [hunk 2]
```diff
@@ -0,0 +1,483 @@
...
+inclusivity
+livestream
+livestreaming
+lockdown
+pandoc
+plaindocs
+polkadots
+recommonmark
+repo
+repo
```
Comment (bethaitman) [inline vale/Vocab/WTD/accept.txt:416]: repeated on previous line

File: vale/Vocab/WTD/accept.txt (hunk 1 above)
Comment (plaindocs) [reply vale/Vocab/WTD/accept.txt:483]: <3

**Pull request closed without merging**